### PR TITLE
added CITATION.cff

### PR DESCRIPTION
**Pull request by ESadek-MO**
Added a citation.cff file to allow users to cite. 

### List of things to do ?post-release?
- [ ] Need to create a list of contributors' orcid id
- [x] Need to change up version on release
- [ ] Need to add doi after release

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- family-names: "Little"
+  given-names: "Bill"
+  orcid: "https://orcid.org/0000-0002-1345-9465"
+- family-names: "Sadek"
+  given-names: "Elias"
+  orcid: "https://orcid.org/0009-0007-3284-9745"
+
+title: "tephi"
+abstract: "Tephigram plotting in Python"
+version: "0.2.4"
```
Comment (bjlittle) [inline CITATION.cff:13]: @ESadek-MO We don't include the version here, otherwise we'll have to constantly update it.

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+date-released: 2025-05-25
+license: "BSD-3-Clause"
+license-url: "https://spdx.org/licenses/BSD-3-Clause.html"
+doi: ""
```
Comment (bjlittle) [inline CITATION.cff:17]: We need a `doi` here, so really can only merge this PR post next release

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  given-names: "Bill"
+  orcid: "https://orcid.org/0000-0002-1345-9465"
+- family-names: "Sadek"
+  given-names: "Elias"
+  orcid: "https://orcid.org/0009-0007-3284-9745"
+
+title: "tephi"
+abstract: "Tephigram plotting in Python"
+version: "0.2.4"
+date-released: 2025-05-25
```
Comment (bjlittle) [inline CITATION.cff:14]: You appear to have brought this file back from the future :stuck_out_tongue_winking_eye:

First release appears to be `"2014-07-01"`

**Review by bjlittle — CHANGES_REQUESTED** [review]
A few comments to service.

Perhaps remove the `doi` entry, which we can add post next release?

File: CITATION.cff (hunk 3 above)
Comment (ESadek-MO) [reply CITATION.cff:14]: Sometimes I confuse myself. Often, in fact.

File: CITATION.cff (hunk 2 above)
Comment (ESadek-MO) [reply CITATION.cff:17]: Yup, will remove for now.

File: CITATION.cff (hunk 1 above)
Comment (ESadek-MO) [reply CITATION.cff:13]: <3

File: CITATION.cff [hunk 4]
```diff
@@ -10,10 +10,8 @@ authors:
 
 title: "tephi"
 abstract: "Tephigram plotting in Python"
-version: "0.2.4"
-date-released: 2025-05-25
+date-released: 2014-07-01
```
[the remark this answers is not in the document]
Comment (ESadek-MO) [reply CITATION.cff:13]: Agh! Yup, apologies.

**Comment by bjlittle** [comment]
@ESadek-MO You could also add the GHA that checks the validity of the `.cff`.

As an example, see https://github.com/bjlittle/geovista/blob/main/.github/workflows/ci-citation.yml

**Review by bjlittle — APPROVED** [review]

**Pull request merged**